airgun/browser.py

```diff
@@ -395,8 +374,7 @@ def __init__(self, name=None, image=None, capabilities=None):
         """
         if docker is None:
             raise DockerBrowserError(
-                'Package docker-py is not installed. Install it in order to '
-                'use DockerBrowser.'
+                'Package docker-py is not installed. Install it in order to ' 'use DockerBrowser.'
```

Comment:
```suggestion
                'Package docker-py is not installed. Install it in order to use DockerBrowser.'
```

Reply:
Found and fixed all of these instances for both single and double quotes, good catch!